src/System.Private.ServiceModel/tests/Scenarios/Contract/XmlSerializer/XmlSerializerFormatTest.cs

```diff
@@ -9,6 +9,46 @@
 using System;
 using System.Reflection;
 using System.Xml.Serialization;
+using System.ServiceModel.Channels;
+using System.ServiceModel.Description;
+using System.ServiceModel.Dispatcher;
+
+
+public class InspectorBehavior : IEndpointBehavior
+{
+    public void AddBindingParameters(ServiceEndpoint endpoint, BindingParameterCollection bindingParameters)
+    {
+
+    }
+
+    public void ApplyClientBehavior(ServiceEndpoint endpoint, ClientRuntime clientRuntime)
+    {
+        clientRuntime.MessageInspectors.Add(new MyMessageInspector());
```

Comment:
This should be `clientRuntime.ClientMessageInspectors.Add(new MyMessageInspector());`

One thing I do when writing scenario tests is to first do it in VS by creating a new .NET Core console app, it's easier to do it there outside of the repos build infrastructure and VS helps you catch errors such as this one.